Cargo.toml

```diff
@@ -649,7 +653,7 @@ walkdir = "2.3.3"
 warp = { version = "0.3.5", features = ["tls"] }
 warp-reverse-proxy = "1.0.0"
 which = "4.2.5"
-x25519-dalek = "1.2.0"
+x25519-dalek = { git = "https://github.com/aptos-labs/x25519-dalek" }
```

Comment:
should we pin to a commit? just to be safer...

Reply:
Done